src/application/gitcoin/utils.ts

```diff
@@ -18,6 +15,7 @@ export const selectTwitterApplications = (
   return twitterApplications;
 };
 
+// TODO: consider using lodash.groupBy
```

Comment:
Not sure if that will work in this case, but there is also a new Object groupBy feature available.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy#examples

Reply:
oh nice1, I didn't know this1! I think we will need to change lib in tsconfig because right now typescript doesn't see this method. I updated TODO for now, is it okay to leave it like that?